libp2p/Host.cpp

```diff
@@ -421,6 +421,18 @@ void Host::onNodeTableEvent(NodeID const& _n, NodeTableEventType const& _e)
     }
 }
 
+bool Host::isConnecting(NodeID const& _id)
```

Comment:
Can it be const method? (make mutex mutable if needed)

Reply:
Maybe it should be more precisely called `isHandshaking`, because connection itself actually already exists during handshake

Reply:
Need to make mutex mutable, will do this and rename the method.

Reply:
@gumb0 Addressed